Regression/WarpX-tests.ini

```diff
@@ -1788,6 +1788,24 @@ analysisRoutine = Examples/Tests/collision/analysis_collision_2d.py
 aux1File = Regression/PostProcessingUtils/post_processing_utils.py
 tolerance = 1.e-14
 
+[collisionRZ]
+buildDir = .
+inputFile = Examples/Tests/collision/inputs_rz
+runtime_params =
+dim = 2
+addToCompileString = USE_RZ=TRUE USE_PSATD=FALSE
+restartTest = 0
+useMPI = 1
+numprocs = 1
+useOMP = 1
+numthreads = 1
+compileTest = 0
+doVis = 0
+compareParticles = 0
+analysisRoutine = Examples/Tests/collision/analysis_collision_rz.py
+aux1File = Regression/PostProcessingUtils/post_processing_utils.py
+tolerance = 1.e-30
```

Comment:
That's stricter than double-precision, is that reasonable? :)

Reply:
When I tested it, I got even smaller than 1e-30, I got something like 1e-36.
I'm not sure, do you want to release it to a bigger tolerance?

Reply:
I wasn't sure if this is a relative or absolute tolerance.
Just checked and it is:
https://github.com/AMReX-Codes/regression_testing/blob/main/test_util.py#L144-L146

If we get a smaller tolerance in comparison out, then this is due to subnormal floating point numbers, which I would not necessarily rely on (e.g., on GPUs a common optimization is to flush subnormal floating point numbers to zero).

Long story short: I would go to set the tolerance to the regular double precision digits (1.e-15).